README.md

```diff
@@ -140,10 +140,9 @@ those reports to your CI system.
 #### CML Reports
 
 The `cml-send-comment` command can be used to post reports. CML reports are
-written in [GitHub Flavored Markdown](https://github.github.com/gfm/). That
-means they can contain images, tables, formatted text, HTML blocks, code
-snippets and more — really, what you put in a CML report is up to you. Some
-examples:
+written in [GitHub Flavored Markdown](https://github.github.com/gfm). That means
```

Comment:
Not sure if this is true for GitLab and BitBucket 😉

Reply:
**See also**

https://github.github.com/gfm
https://docs.gitlab.com/ee/user/markdown.html
https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html

Reply:
@casperdcl, we support three platforms, and all of them seem to follow slightly different standards: 🤔 

https://github.github.com/gfm
https://docs.gitlab.com/ee/user/markdown.html
https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html